ts/components/screens/ListItemComponent.tsx

```diff
@@ -0,0 +1,106 @@
+import { ListItem, Text, View } from "native-base";
+import * as React from "react";
+import { StyleSheet } from "react-native";
+import customVariables from "../../theme/variables";
+import H5 from "../ui/H5";
+import IconFont from "./../ui/IconFont";
+import { BadgeComponent } from "./BadgeComponent";
+
+type Props = Readonly<{
+  title: string;
+  subTitle: string;
+  onPress: () => void;
+  isFirstItem?: boolean;
+  isLastItem?: boolean;
+  hasBadge?: boolean;
+  iconName?: string;
+  hideIcon?: boolean;
+  useExtendedSubTitle?: boolean;
+}>;
+
+const ICON_SIZE = 24;
+
+const styles = StyleSheet.create({
+  listItem: {
+    paddingLeft: 0,
+    paddingRight: 0,
+    paddingTop: customVariables.spacerHeight,
+    paddingBottom: customVariables.spacerHeight
+  },
+  spacingBase: {
+    paddingRight: customVariables.spacingBase
+  },
+  flexRow: {
+    flexDirection: "row",
+    justifyContent: "space-between"
+  },
+  serviceName: {
+    fontWeight: "700"
+  },
+  description: {
+    paddingRight: ICON_SIZE,
+    alignSelf: "flex-start"
+  },
+  noBorder: {
+    borderBottomWidth: 0
+  }
+});
+
+export default class ListItemComponent extends React.Component<Props> {
+  public render() {
+    return (
+      <ListItem
+        style={[
+          styles.listItem,
+          this.props.isLastItem && styles.noBorder,
```

Comment:
Use the "last" prop.

Reply:
If used, last prop don't work as expected (it does not hide the bottom item separator)